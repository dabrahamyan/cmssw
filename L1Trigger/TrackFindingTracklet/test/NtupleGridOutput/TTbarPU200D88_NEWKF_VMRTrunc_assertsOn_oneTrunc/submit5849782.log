000 (5849782.000.000) 09/07 10:44:45 Job submitted from host: <137.138.44.55:9618?addrs=137.138.44.55-9618+[2001-1458-d00-1a--48]-9618&alias=bigbird17.cern.ch&noUDP&sock=schedd_7840_2a97>
...
040 (5849782.000.000) 09/07 10:52:25 Started transferring input files
	Transferring to host: <128.142.137.84:9618?addrs=128.142.137.84-9618+[2001-1458-301-5e--100-48]-9618&alias=b7g19p7086.cern.ch&noUDP&sock=slot1_4_4089_d1ff_3874>
...
040 (5849782.000.000) 09/07 10:52:25 Finished transferring input files
...
001 (5849782.000.000) 09/07 10:52:26 Job executing on host: <128.142.137.84:9618?addrs=128.142.137.84-9618+[2001-1458-301-5e--100-48]-9618&alias=b7g19p7086.cern.ch&noUDP&sock=startd_3497_3ac7>
...
006 (5849782.000.000) 09/07 10:52:35 Image size of job updated: 35956
	32  -  MemoryUsage of job (MB)
	32592  -  ResidentSetSize of job (KB)
...
006 (5849782.000.000) 09/07 10:57:36 Image size of job updated: 2427544
	1034  -  MemoryUsage of job (MB)
	1058308  -  ResidentSetSize of job (KB)
...
006 (5849782.000.000) 09/07 11:12:37 Image size of job updated: 2591384
	1034  -  MemoryUsage of job (MB)
	1058308  -  ResidentSetSize of job (KB)
...
006 (5849782.000.000) 09/07 11:17:37 Image size of job updated: 2600012
	1034  -  MemoryUsage of job (MB)
	1058308  -  ResidentSetSize of job (KB)
...
006 (5849782.000.000) 09/07 11:22:37 Image size of job updated: 2600012
	1042  -  MemoryUsage of job (MB)
	1066076  -  ResidentSetSize of job (KB)
...
006 (5849782.000.000) 09/07 11:37:39 Image size of job updated: 2600012
	1043  -  MemoryUsage of job (MB)
	1067908  -  ResidentSetSize of job (KB)
...
006 (5849782.000.000) 09/07 11:44:38 Image size of job updated: 6463156
	1043  -  MemoryUsage of job (MB)
	1067908  -  ResidentSetSize of job (KB)
...
040 (5849782.000.000) 09/07 11:44:38 Started transferring output files
...
040 (5849782.000.000) 09/07 11:44:38 Finished transferring output files
...
005 (5849782.000.000) 09/07 11:44:38 Job terminated.
	(1) Normal termination (return value 139)
		Usr 0 00:45:24, Sys 0 00:01:31  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:45:24, Sys 0 00:01:31  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	74581  -  Run Bytes Sent By Job
	697  -  Run Bytes Received By Job
	74581  -  Total Bytes Sent By Job
	697  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     1.28        1         1 
	   Disk (KB)            :    94           1   1556904 
	   Memory (MB)          :  1043        2000      2000 

	Job terminated of its own accord at 2023-09-07T09:44:38Z.
...
