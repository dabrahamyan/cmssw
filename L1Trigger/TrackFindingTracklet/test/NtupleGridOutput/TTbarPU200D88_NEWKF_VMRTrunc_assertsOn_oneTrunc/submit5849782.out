using geometry D88 (tilted)
Thu Sep  7 11:44:01 CEST 2023
Thread 7 (Thread 0x2afa49c4e700 (LWP 184) "cmsRun"):
#0  0x00002afa160eab3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002afa160eabcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002afa160eac6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002afa1c4ca576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002afa1c4ca639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002afa160e4ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002afa163f7b0d in clone () from /lib64/libc.so.6
Thread 6 (Thread 0x2afa49a4d700 (LWP 183) "cmsRun"):
#0  0x00002afa160eab3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002afa160eabcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002afa160eac6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002afa1c4ca576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002afa1c4ca639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002afa160e4ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002afa163f7b0d in clone () from /lib64/libc.so.6
Thread 5 (Thread 0x2afa4984c700 (LWP 182) "cmsRun"):
#0  0x00002afa160eab3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002afa160eabcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002afa160eac6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002afa1c4ca576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002afa1c4ca639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002afa160e4ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002afa163f7b0d in clone () from /lib64/libc.so.6
Thread 4 (Thread 0x2afa4964b700 (LWP 181) "cmsRun"):
#0  0x00002afa160ebe9d in nanosleep () from /lib64/libpthread.so.0
#1  0x00002afa1c20710f in XrdSysTimer::Wait(int) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#2  0x00002afa1c4382d3 in XrdCl::TaskManager::RunTasks() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#3  0x00002afa1c438429 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002afa160e4ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002afa163f7b0d in clone () from /lib64/libc.so.6
Thread 3 (Thread 0x2afa4944a700 (LWP 180) "cmsRun"):
#0  0x00002afa163f80e3 in epoll_wait () from /lib64/libc.so.6
#1  0x00002afa1c2015c2 in XrdSys::IOEvents::PollE::Begin(XrdSysSemaphore*, int&, char const**) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#2  0x00002afa1c1fdd4d in XrdSys::IOEvents::BootStrap::Start(void*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#3  0x00002afa1c206818 in XrdSysThread_Xeq () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#4  0x00002afa160e4ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002afa163f7b0d in clone () from /lib64/libc.so.6
Thread 2 (Thread 0x2afa3a55f700 (LWP 173) "cmsRun"):
#0  0x00002afa160ec1d9 in waitpid () from /lib64/libpthread.so.0
#1  0x00002afa1cb27ad7 in edm::service::cmssw_stacktrace_fork() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#2  0x00002afa1cb286ca in edm::service::InitRootHandlers::stacktraceHelperThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#3  0x00002afa15cc0f90 in std::execute_native_thread_routine (__p=0x2afa28ce2e30) at ../../../../../libstdc++-v3/src/c++11/thread.cc:80
#4  0x00002afa160e4ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002afa163f7b0d in clone () from /lib64/libc.so.6
Thread 1 (Thread 0x2afa1847ae40 (LWP 155) "cmsRun"):
#0  0x00002afa163ecddd in poll () from /lib64/libc.so.6
#1  0x00002afa1cb27dcf in full_read.constprop () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#2  0x00002afa1cb2879c in edm::service::InitRootHandlers::stacktraceFromThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#3  0x00002afa1cb2b24b in sig_dostack_then_abort () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#4  <signal handler called>
#5  0x00002afa46e953b5 in std::_Sp_counted_base<(__gnu_cxx::_Lock_policy)2>::_M_release() () from /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginTrackFindingTrackletPlugins.so
#6  0x00002afa46ec6386 in trklet::ProducerKFout::produce(edm::Event&, edm::EventSetup const&) () from /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginTrackFindingTrackletPlugins.so
#7  0x00002afa13d0d1e3 in edm::stream::EDProducerAdaptorBase::doEvent(edm::EventTransitionInfo const&, edm::ActivityRegistry*, edm::ModuleCallingContext const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#8  0x00002afa13cefc3f in edm::WorkerT<edm::stream::EDProducerAdaptorBase>::implDo(edm::EventTransitionInfo const&, edm::ModuleCallingContext const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#9  0x00002afa13c3fb25 in decltype ({parm#1}()) edm::convertException::wrap<edm::Worker::runModule<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*)::{lambda()#1}>(edm::Worker::runModule<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*)::{lambda()#1}) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#10 0x00002afa13c3fe13 in std::__exception_ptr::exception_ptr edm::Worker::runModuleAfterAsyncPrefetch<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(std::__exception_ptr::exception_ptr, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#11 0x00002afa13c42546 in edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#12 0x00002afa139db879 in tbb::detail::d1::function_task<edm::WaitingTaskList::announce()::{lambda()#1}>::execute(tbb::detail::d1::execution_data&) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreConcurrency.so
#13 0x00002afa154d5bec in tbb::detail::r1::task_dispatcher::local_wait_for_all<false, tbb::detail::r1::external_waiter> (waiter=..., t=0x2afa19c39f00, this=0x2afa19c67700) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.h:322
#14 tbb::detail::r1::task_dispatcher::local_wait_for_all<tbb::detail::r1::external_waiter> (waiter=..., t=<optimized out>, this=0x2afa19c67700) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.h:463
#15 tbb::detail::r1::task_dispatcher::execute_and_wait (t=<optimized out>, wait_ctx=..., w_ctx=...) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.cpp:168
#16 0x00002afa13bba22d in edm::FinalWaitingTask::wait() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#17 0x00002afa13ba2310 in edm::EventProcessor::processRuns() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#18 0x00002afa13baeda1 in edm::EventProcessor::runToCompletion() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#19 0x000000000040a266 in tbb::detail::d1::task_arena_function<main::{lambda()#1}::operator()() const::{lambda()#1}, void>::operator()() const ()
#20 0x00002afa154c40eb in tbb::detail::r1::task_arena_impl::execute (ta=..., d=...) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/arena.cpp:698
#21 0x000000000040b094 in main::{lambda()#1}::operator()() const ()
#22 0x000000000040971c in main ()
